xbmc/filesystem/VideoDatabaseDirectory/DirectoryNodeMovieAssetTypes.cpp

```diff
@@ -18,5 +21,18 @@ CDirectoryNodeMovieAssetTypes::CDirectoryNodeMovieAssetTypes(const std::string&
 
 NodeType CDirectoryNodeMovieAssetTypes::GetChildType() const
 {
-  return NodeType::MOVIE_ASSETS;
+  CQueryParams params;
+  CollectQueryParams(params);
+
+  switch (params.GetVideoAssetType())
+  {
+    case -2: // special value for all assets + extras virtual folder
```

Comment:
magic number ... and you even wrote a comment what it represents (iirc, not for the first time in this PR) ;-)